charts/linkerd2/templates/identity.yaml

```diff
@@ -21,6 +21,19 @@ data:
   key.pem: {{b64enc (required "Please provide the identity issue private key" .Values.identity.issuer.tls.keyPEM | trim)}}
 {{- end}}
 ---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: linkerd-identity-trust-roots
+  namespace: {{.Values.namespace}}
+  labels:
+    linkerd.io/control-plane-component: identity
+    linkerd.io/control-plane-ns: {{.Values.namespace}}
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+data:
+  ca.pem: |{{.Values.identityTrustAnchorsPEM | trim | nindent 4}}
```

Comment:
We'll probably want a way to skip adding this resource, as we do for the issuer secret:

https://github.com/linkerd/linkerd2/blob/a0c423f55b8e200b1a175f3d9a0715ff44e91a1b/charts/linkerd2/templates/identity.yaml#L6

We probably don't want to use the same mechanism though, or if we do we'll need to document how to create the configmap... perhaps its worth introducing a new flag--something like `.Values.identity.externalCA`--to control both this and the issuer secret?

Reply:
nit: let's call the key `ca-bundle.crt`, which matches OS configuration like `/etc/ssl/certs/ca-bundle.crt`